tests/specs/csp.e2e.js

```diff
@@ -40,17 +43,30 @@ describe('Content Security Policy', () => {
   })
 
   it('should load async chunk with subresource integrity', async () => {
+    await browser.enableSessionReplay()
+
+    const url = await browser.testHandle.assetURL('subresource-integrity-capture.html', {
+      init: {
+        privacy: { cookies_enabled: true },
+        session_replay: { enabled: true, sampling_rate: 100, error_sampling_rate: 100 }
+      }
+    })
     await Promise.all([
       browser.testHandle.expectRum(),
-      browser.url(await browser.testHandle.assetURL('subresource-integrity-capture.html'))
+      browser.url(url)
         .then(() => browser.waitForAgentLoad())
     ])
 
+    await browser.waitUntil(
+      () => browser.execute(function () {
+        return window.chunkIntegratyValues.length === 3
+      })
+    )
     const foundIntegrityValues = await browser.execute(function () {
       return window.chunkIntegratyValues
     })
 
-    expect(foundIntegrityValues.length).toEqual(1)
+    expect(foundIntegrityValues.length).toEqual(3)
```

Comment:
This expect becomes redundant because execution will only pass the previous lines of code if the number of values hits 3.